templates/templates/preview/activeinfra_html.template

```diff
@@ -0,0 +1,14 @@
+<ul class="entityList">      
+        <li>= Database identifier: {{ db.name }}</li>  
+            <li>Shards: {{ db.num_shards }}</li>
+            <li>Network interface: {{ db._interfaces[0] }}</li>
+            <li>Type: {% set db_type, _ = config.database_cli.split("/")[-1].split("-",1) %}{{ db_type }}</li>
+            <li>TCP/IP port:
+            <ul class="entityList">
+                {%- for port in db.ports %}
+                <li>{{ port }}</li>
+                {%- endfor %}
+            </ul>
+            </li>
+            <li>=</li>
+</ul>   
```

Comment:
trailing newline to appease the git monster

Reply:
added